packages/client/src/views/OfficeHome/readyForReview/ReadyForReview.tsx

```diff
@@ -101,10 +101,17 @@ class ReadyForReviewComponent extends React.Component<
   }
 
   componentDidMount() {
+    window.addEventListener('popstate', this.popStateListener)
+
     window.addEventListener('resize', this.recordWindowWidth)
   }
 
+  popStateListener() {
+    window.history.pushState(null, document.title, window.location.href)
+  }
+
   componentWillUnmount() {
+    window.addEventListener('popstate', this.popStateListener)
```

Comment:
@favour234 I dont like this solution.  You add a listener on mount and unmount.  What is wrong with how you approached https://github.com/opencrvs/opencrvs-core/pull/4050 ?

Why are you removing code in Header.tsx that you added in that PR? ``` (REGISTRAR_ROLES.includes(role as string) &&
        location.includes(REGISTRAR_HOME))```

Reply:
the code "(REGISTRAR_ROLES.includes(role as string) && location.includes(REGISTRAR_HOME))" resolved the issue but unknowingly created another which @CornelMaze  worked on.

Reply:
> @favour234 I dont like this solution. You add a listener on mount and unmount. What is wrong with how you approached #4050 ?
> 
> Why are you removing code in Header.tsx that you added in that PR? ` (REGISTRAR_ROLES.includes(role as string) && location.includes(REGISTRAR_HOME))`



> @favour234 I dont like this solution. You add a listener on mount and unmount. What is wrong with how you approached #4050 ?
> this code was also added in my previous pr https://github.com/opencrvs/opencrvs-core/pull/4050/files.
After redirecting to home page on correction or printing of a certificate, its observed that we still had access to go back from chromes  browser back button, which result to system crashing , So the popStateListener  is basically to change the history to the present state, which is the present location of the page..so if you click on the back button on the browser ..you still remain on that page.  its a little function to replace  windows history to the present window's location.
 
 
> Why are you removing code in Header.tsx that you added in that PR? ` (REGISTRAR_ROLES.includes(role as string) && location.includes(REGISTRAR_HOME))`

Reply:
@favour234 You need `removeEventListener` in unmount and not add